docs/content/latest/architecture/transactions/isolation-levels.md

```diff
@@ -16,31 +16,34 @@ showAsideToc: true
 
 Transaction isolation is foundational to handling concurrent transactions in databases. The [SQL-92 standard](https://en.wikipedia.org/wiki/SQL-92) defines four levels of transaction isolation, `SERIALIZABLE`, `REPEATABLE READ`, `READ COMMITTED` and `READ UNCOMMITTED` in decreasing order of strictness.
 
-YugabyteDB supports two transaction isolation levels - `SERIALIZABLE` (which maps to the SQL isolation level of the same name) and `SNAPSHOT` (which is mapped to the SQL isolation level `REPEATABLE READ`). Thus, YugabyteDB supports the two strictest of the above four isolation levels. These isolation levels are described below:
+YugabyteDB supports three transaction isolation levels - `READ COMMITTED`, `SERIALIZABLE` (both map to the SQL isolation level of the same name) and `SNAPSHOT` (which is mapped to the SQL isolation level `REPEATABLE READ`). Thus, YugabyteDB supports the three strictest of the above four isolation levels. These isolation levels are described below:
```

Comment:
@pkj415 Not sure where to exactly make the change but we need to call out that READ COMMITTED is in beta and not enabled by default. And possibly call out the flag to enable it too. Otherwise this will confuse users running default YB and getting the wrong semantics (until we actually change the default -- then we'd updates the docs again).